library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -79,8 +81,8 @@ task BuildCellRangerReference {
 
     # The reference building process follows the instructions from
     # https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references
-
-    cellranger mkgtf "$gtf_filename" "filtered.$gtf_filename" \
+    if [[ "~{ref_source}"=="Ensembl" ]]; then
```

Comment:
If the parameters to the cellranger mkgtf call will only vary by the attribute type, you could do something like this to consolidate it a bit more:

```
if [[ "~{ref_source}"=="Ensembl" ]]; then
     attribute_type="gene_biotype"
elif [[ "~{ref_source}"=="Gencode" ]]; then
     attribute_type="gene_type"
fi
cellranger mkgtf "$gtf_filename" "filtered.$gtf_filename" \
              --attribute=${attribute_type}:protein_coding \
              --attribute=${attribute_type}:lincRNA \
              ....
```

Reply:
Oh, okay! That makes sense to leave it as-is then

Reply:
@jishuxu can you please make the indentation consistent here?

Reply:
is it better now?

Reply:
Yes, thank you!